rust/src/delta.rs

```diff
@@ -672,6 +689,15 @@ impl DeltaTable {
         self.get_files_iter().collect()
     }
 
+    /// Returns file names present in the loaded state in HashSet
+    pub fn get_file_set(&self) -> HashSet<&str> {
```

Comment:
nitpick: since we are returning file paths, we should rename the method, proposition:
```suggestion
    pub fn get_file_paths_set(&self) -> HashSet<&str> {
```

Reply:
file path is defined as `self.storage.join_path(&self.table_path, &add.path` right? this method returns `add.path`.